samples/bluetooth/mesh/prj.conf

```diff
@@ -8,7 +8,6 @@ CONFIG_SETTINGS=y
 CONFIG_HWINFO=y
 
 CONFIG_BT=y
-CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
Should it be removed for microbit_gatt.conf? Then microbit will also access controller function directly.